### PR TITLE
Fix BuildTool output path

### PR DESCRIPTION
**Pull request by hazzik**
There seems to be a difference in `-o` behaviour between versions of .NET Sdk. The .NET Sdk up to 2.2 output the build artefacts relative to the project file, and starting from previews of 3.0 the build output is calculated relative to the current directory.

### CODE REVIEW

**Comment by hazzik** [comment]
@lillo42 please test.

**Comment by lillo42** [comment]
@hazzik it is not fixes yet:

```
rlillo@Lenovo-ideapad-320:~/Downloads/nhibernate-core-fix-building-build-tool$ ./ShowBuildMenu.sh 
./ShowBuildMenu.sh: 17: ./ShowBuildMenu.sh: pushd: not found
Microsoft (R) Build Engine version 16.3.0-preview-19455-02+4a2d77107 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1009: Project file does not exist.
Switch: BuildTool.sln
./ShowBuildMenu.sh: 19: ./ShowBuildMenu.sh: popd: not found
========================= NHIBERNATE BUILD MENU ==========================
--- TESTING ---
A. (Step 1) Set up a new test configuration for a particular database.
B. (Step 2) Activate a test configuration.
C. (Step 3) Run tests.
.
--- BUILD ---
E. Build NHibernate (Debug)
F. Build NHibernate (Release)
.
--- Code generation ---
H. Generate async code (Generates files in Async sub-folders)
.
--- Exit ---
X. Make the beautiful build menu go away.
.
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET Core program, but dotnet-./Tools/BuildTool/bin/BuildTool.dll does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET Core program, but dotnet-./Tools/BuildTool/bin/BuildTool.dll does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
cat: /home/rlillo/Downloads/nhibernate-core-fix-building-build-tool/folder.tmp: No such file or directory
cp: cannot access '/lost+found': Permission denied
```

**Comment by hazzik** [comment]
> pushd: not found

Very strange...

**Comment by hazzik** [comment]
@lillo42 I've replaced pushd/popd with cd. Could you please check now?

**Comment by lillo42** [comment]
@hazzik it's working now \o/

File: ShowBuildMenu.sh [hunk 1]
```diff
@@ -14,8 +15,10 @@ async_generator_path=""
 
 if [ ! -f $BUILD_TOOL_PATH ]
 then
-	dotnet build ./Tools/BuildTool/BuildTool.sln -c Release -o bin
+	cd ./Tools/BuildTool
+	dotnet build BuildTool.sln -c Release -o bin 
 fi
+cd "$SCRIPT_PATH"
```
Comment (fredericDelaporte) [inline ShowBuildMenu.sh]: I would just have used `cd ../..` before the `fi`, but why not.

**Comment by fredericDelaporte** [comment]
(Verified under Windows and Linux, from the same folder and from `src`, with and without the build tool already built, but without dotnet 3.)

File: ShowBuildMenu.sh (hunk 1 above)
Comment (hazzik) [reply ShowBuildMenu.sh]: I was thinking, but decided against it.

**Pull request merged**